---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

[PDF version](http://zhiyhu.github.io/files/Zhiyuan_CV.pdf)

Education
======
* B.S. in Biological Science, Peking University, China, 2015
* D.Phil in Clinical Medicine, University of Oxford, United Kingdom, 2020


Awards & Grants
======
WHG Public Engagement Seed Award, University of Oxford (£1,500), 2019  
NIHR Oxford Biomedical Research Centre Small Grants in Health Sciences (£24,395), 2019  
AACR-Margaret Foti Scholar-in-Training Award, AACR ($2,000), 2019  
Travel and Research Fund, St Cross College, University of Oxford (£500), 2019  
Poster Prize, CRUK Oxford Centre 2018 Symposium, 2018  
[NDM Graduate Student Prize](https://www.ndm.ox.ac.uk/zhiyuan-hu), University of Oxford, 2017  
Shen Tong Outstanding Undergraduate Award, Peking University (CNY 20000), 2015  
China National Scholarship (CNY 8000), 2014  
Li & Fung Scholarship, Victor and William Fung Foundation, 2014  
Arawana Scholarship, Yihai Kerry, Wilmar China (CNY 12000), 2013  
Robin Li Scholarship, Robin Li Foundation (CNY 5000), 2012  


Skills
======
**Web lab skills**
* Tissue processing: human/mouse sample dissociation, primary cell preservation, primary cell culture and etc.
* Next-generation sequencing: RNA-seq library preparation, single-cell RNA-seq (Smart-seq2 and 10x), Illumina sequencing and etc.
* Cell biology techniques: fluorescent staining, flow cytometry/FACS, confocal microscope, cell culture and etc.
* Molecular techniques: DNA/RNA extraction, RT, PCR, qPCR, electrophoresis, Sanger sequence, NanoString assay and etc.

**Data analysis skills**
* Proficient in: R (daily usage), Shell (regular usage), Git (daily usage), Python (Snakemake incl., regular usage), analysis of RNA/DNA-seq data, single-cell RNA-seq data and whole-genome methylation data
* Familiar with: C (used extensively during undergrad), MATLAB (used for a undergrad course project)
* Familiar with statistical learning and Bayesian inference.

**Other relevent**
* Proficient in LaTeX, Illustrator, iMovie and etc.


Supervision experience
======
Supervising summer intern, 2017  
Supervising summer intern, 2019  
Supervising predoctoral research assistant, 2020  


Undergrad research experience
======
**Summer intern**, Nuffield Department of Medicine, University of Oxford, 2014  
Analysis of the metformin effect on cancer single-cell transcriptome *Advisors: Dr. Chris Yau and Dr. Quin Wills*

**Research assistant**, Center for Quantitative Biology, Peking University, 2014-2015  
Phylogenetic analysis of horizontal gene transfer in microbial microevolution. *Advisor: Prof Huaiqiu Zhu*

**Research assistant**, Center for Quantitative Biology, Peking University, 2013-2014  
Analysis of non-coding regions in horizontal gene transfer. *Advisor: Prof Huaiqiu Zhu*


Undergrad background in math/programming/compbio
======
Advanced Mathematics (B)(1), 96/100  
Advanced Mathematics (B)(2), 98/100  
Linear Algebra (B), 93/100  
Probability Theory and Statistics (B), 90/100  
Applied Stochastic Models, 99/100  
Data Structure and Algorithm (B), 96/100  
Mathematical Modeling in the Life Sciences, 90/100  
Methods in Bioinfomatics, 94/100  
Mathematical Biophysics, 100/100  
  
*(B) type cources are for students majoring in science.*

